src/server/job_recov_db.c

```diff
@@ -690,14 +694,17 @@ resv_recov_db(char *resvid)
 		goto db_err;
 
 	if (db_to_svr_resv(presv, &dbresv) != 0)
-		goto db_err;
+		goto db_err_reset;
 
 	pbs_db_reset_obj(&obj);
 
 	if (pbs_db_end_trx(conn, PBS_DB_COMMIT) != 0)
 		goto db_err;
 
 	return (presv);
+
+db_err_reset:
```

Comment:
Same comment as above

Reply:
Fixed in https://github.com/PBSPro/pbspro/pull/1394/commits/019f046bf1570332e453e9d6451c79abfe7d5bb7